gamemode/README.md

```diff
@@ -0,0 +1,5 @@
+This is a submodule for FeralInteractive's GameMode.
+
+To use this, add the manifests listed here; `libgamemode*.json` will install the shared libraries, `gamemoderun.json` will install `gamemoderun` to `/app/bin`.
+
+You will need to add both manifests separately.
```

Comment:
What's the harm in `libgamemode` always installing `gamemoderun`?

Reply:
with libgamemode, apparently the meson buildsystem doesn't install gamemoderun to the prefix, no matter the config-opts. I can make an issue in upstream gamemode about it, or see if I can find something about why it's not done.

Reply:
I'll merge the two manifests, it's not a large deal to add a separate module for gamemoderun anyways.

Reply:
You can add a custom `"post-install"` command to install the script also.

Reply:
Looking at their meson files they correctly install it into the right prefix..

Reply:
It was fixed in https://github.com/FeralInteractive/gamemode/commit/85d75c303c61d75602a8415716aa9d4befbf6f8a

Reply:
> Looking at their meson files they correctly install it into the right prefix..

Interesting, it didn't do that when I built it without the config-opts. Is there anything I need to do to get it to install?

Reply:
It is behind the `if sd_bus_provider != 'no-daemon'` option.

Reply:
I think their logic is just wrong. A manual install seems fine.